test/unit/event_machine_proxy_test.rb

```diff
@@ -3,10 +3,27 @@
 describe ExceptionalSynchrony::EventMachineProxy do
   include TestHelper
 
+  class RunProxyMock
+    def self.run(&block)
```

Comment:
Can you use `class << self` throughout? That's now our preferred way instead of `self.`.